zebedee-cms/src/main/java/com/github/onsdigital/zebedee/authorisation/AuthorisationServiceImpl.java

```diff
@@ -0,0 +1,63 @@
+package com.github.onsdigital.zebedee.authorisation;
+
+import com.github.onsdigital.zebedee.api.Root;
+import com.github.onsdigital.zebedee.service.ServiceSupplier;
+import com.github.onsdigital.zebedee.session.model.Session;
+import com.github.onsdigital.zebedee.session.service.SessionsService;
+import com.github.onsdigital.zebedee.user.service.UsersService;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.IOException;
+
+import static com.github.onsdigital.zebedee.logging.ZebedeeLogBuilder.logError;
+import static com.github.onsdigital.zebedee.logging.ZebedeeLogBuilder.logWarn;
+import static org.apache.http.HttpStatus.SC_INTERNAL_SERVER_ERROR;
+import static org.apache.http.HttpStatus.SC_NOT_FOUND;
+import static org.apache.http.HttpStatus.SC_UNAUTHORIZED;
+
+public class AuthorisationServiceImpl implements AuthorisationService {
+
+	private ServiceSupplier<SessionsService> sessionServiceSupplier = () -> Root.zebedee.getSessionsService();
+	private ServiceSupplier<UsersService> userServiceSupplier = () -> Root.zebedee.getUsersService();
+
+	@Override
+	public UserIdentity identifyUser(String sessionID) throws UserIdentityException {
+		if (StringUtils.isEmpty(sessionID)) {
+			logWarn("identify user error, no auth token was provided").log();
+			throw new UserIdentityException("user not authenticated", SC_UNAUTHORIZED);
+		}
+
+		Session session;
+		try {
+			session = sessionServiceSupplier.getService().get(sessionID);
+		} catch (IOException e) {
+			logError(e, "identify user error, unexpected error while attempting to get user session")
+					.sessionID(sessionID)
+					.log();
+			throw new UserIdentityException("internal server error", SC_INTERNAL_SERVER_ERROR);
```

Comment:
could the "internal server error" string a constant. Same for the other error messages